core/src/main/java/com/homeaway/streamplatform/streamregistry/exceptions/StreamDeletionException.java

```diff
@@ -18,7 +18,7 @@
 public class StreamDeletionException extends BaseStreamException {
     private static final long serialVersionUID = -5793888606557229357L;
 
-    public StreamDeletionException(String streamName) {
-        super(streamName);
```

Comment:
This should not be `streamName` this should be `msg` and we should fix everything to have the appropriate _*msg*_!!! 😄